app/org/sagebionetworks/bridge/validators/StudyValidator.java

```diff
@@ -13,9 +13,14 @@
 import org.springframework.validation.Errors;
 import org.springframework.validation.Validator;
 
+import com.newrelic.agent.deps.com.google.common.base.Joiner;
```

Comment:
I think there's a merge error here. Pretty sure we removed new relic from our codebase.